librz/util/str.c

```diff
@@ -618,35 +618,35 @@ RZ_API int rz_str_char_count(const char *string, char ch) {
 	return count;
 }
 
-static const char *separator_get_first(const char *text) {
+static const char *skip_non_separator_chars(const char *text) {
 	for (; *text && !IS_SEPARATOR(*text); text++)
 		;
 	;
 
 	return text;
 }
 
-static const char *word_get_first(const char *text) {
+static const char *skip_separator_chars(const char *text) {
 	for (; *text && IS_SEPARATOR(*text); text++)
 		;
 	;
 
 	return text;
 }
 
-RZ_API char *rz_str_word_get_first(const char *text) {
-	return strdup(word_get_first(text));
+RZ_API char *rz_str_skip_separator_chars(const char *text) {
+	return strdup(skip_separator_chars(text));
 }
 
 // Counts the number of words (separated by separator characters: newlines, tabs,
```

Comment:
Could you please convert this description to the Doxygen while you are on it?

 https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#documentation